tests/scripts/analyze_outcomes.py

```diff
@@ -542,28 +542,31 @@ def do_analyze_driver_vs_reference(results: Results, outcomes: Outcomes, args) -
             'ignored_suites': [
                 # Skipped in the accelerated component
                 'aes', 'aria', 'camellia',
-                # These require AES_C and CAMELLIA_C to be enabled in order for the cipher
-                # module (actually cipher_wrapper) to work properly. However these symbols
-                # are disabled in the accelerated component so we ignore them.
-                'cipher.ccm', 'cipher.gcm', 'cmac',
+                # These require AES_C, ARIA_C or CAMELLIA_C to be enabled in
+                # order for the cipher module (actually cipher_wrapper) to work
+                # properly. However these symbols are disabled in the accelerated
+                # component so we ignore them.
+                'cipher.ccm', 'cipher.gcm', 'cmac', 'cipher.aes', 'cipher.aria',
```

Comment:
Ok, so here I'm going to make the comment I originally made on another PR, but this time hopefully it will make sense :) We have built-in DES, built-in CMAC, and no AES. So we shouldn't have to ignore the entire CMAC test suite: only the cases that use AES (those that use DES should work).

I suggest renaming test cases in `test_suite_cmac.data` so that all test using AES have AES in their name, then below we can just do a regex-ignore based on AES.

Reply:
Also, regarding `cipher.aes`, `cipher.gcm`, `cipher.ccm` - they have "PSA" test cases which I expect should be working here. I suggest we create a new test suite `cipher.psa.data` and move all the "PSA" test cases from the above three to the new test suites.

If I'm not mistaken, we should not have to ignore `cipher.psa` here.

Reply:
> I suggest renaming test cases in `test_suite_cmac.data` so that all test using AES have AES in their name, then below we can just do a regex-ignore based on AES.

This is something I did initially, but then I realized that skipping tests with AES, ARIA and CAMELLIA (because also these 2 are used there as check) then basically only 2 items remained, so I eventually skipped the entire suite :)

Reply:
> This is something I did initially, but then I realized that skipping tests with AES, ARIA and CAMELLIA (because also these 2 are used there as check) then basically only 2 items remained, so I eventually skipped the entire suite :)

Ah, right, I had only checked that there were DES tests, not how many. Indeed 2 is not a lot, but I think it's slightly cleaner this way - we are now testing the assertion we're making in the documentation that CMAC works with whatever cipher is built-in.

Reply:
> These PSA tests are actually skipped because `MBEDTLS_DEPRECATED_REMOVED` is disabled. There is a comment in `common_block_cipher_dispatch()` with which I tried to explain the reason. Is it wrong?

Ah, no that's not wrong at all, I just forgot, sorry. Let's not worry about it then.